ops/services/web_application_firewall/main.tf

```diff
@@ -179,36 +179,54 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
        */
       rule_group_override {
         rule_group_name = "REQUEST-920-PROTOCOL-ENFORCEMENT" //TODO: add exception for whoami
```

Comment:
is the TODO still relevant?

Reply:
The todo comment has been deleted. 

That exception was put in place with the rules under the 932 subset. [The GraphQL refactor would be next.](https://github.com/CDCgov/prime-simplereport/issues/3738) Once that is done, we can start working on removing these exceptions.